examples/azure-vm-simple-linux-managed-disk/terraform.tfvars

```diff
@@ -0,0 +1,8 @@
+# Replace with relevant values
+
+# resource_group = "myresourcegroup"
+# rg_prefix      = "rg"
+# hostname       = "myvm"
+# dns_name       = "mydnsname"
+# location       = "southcentralus"
+# admin_password = "T3rr@f0rmP@ssword"
```

Comment:
Would it be possible to set these as defaults in the `variables.tf` file and then remove this file?

It's also worth mentioning if there's no default in Code, the user will be prompted at runtime - which might be a better fit for the sensitive variables e.g. `admin_password`?